src/components/form/text-input/index.tsx

```diff
@@ -61,28 +64,41 @@ export const TextInput = React.forwardRef<HTMLDivElement, TextInputProps>(
         iconBefore={iconBefore}
         {...wrapperProps}
       >
-        {title && (
-          <Text weight={theme.typography.weights.bold} size={theme.typography.sizes.m} className="inputTitle">
-            {title}
-          </Text>
-        )}
-        {subTitle && (
-          <Text size={theme.typography.sizes.xs} className="inputSubtitle">
-            {subTitle}
-          </Text>
-        )}
+        <div className='titleContainer'>
+          <div className={`${subtitle ? 'mb-4' : ''}`}>
```

Comment:
just add `margin-bottom: 4px` to `titleContainer` class

Reply:
In fact, margin-bottom should be a temporary property that depends on the availability of subtitle

Reply:
@doichev-kostia 

```
<div className=`titleContainer ${subtitle ? 'mb-4' : ''}`>
```

Reply:
@lazzystep, if I write template string without curly braces, I'll catch a syntax error

Reply:
Yeah, agree, add them
you get the point about Vlad's message